backend/src/main/java/com/handong/rebon/review/domain/Review.java

```diff
@@ -39,7 +41,7 @@
     private ReviewScore reviewScore;
 
     @OneToMany(mappedBy = "review")
-    private List<Empathy> empathies;
+    private List<Empathy> empathies = new ArrayList<>();
 
     public String getTitle() {
         return getReviewContent().getTitle();
```

Comment:
reviewContent같은 경우는 Review의 필드인데 바로 접근하지 않고 getter를 사용하신 이유가 있나요?
```suggestion
    public String getTitle() {
        return reviewContent.getTitle();
```